components/style/properties/longhand/box.mako.rs

```diff
@@ -1175,9 +1173,9 @@ ${helpers.predefined_type("scroll-snap-coordinate",
                 m32: self.m32.to_computed_value(context),
                 m33: self.m33.to_computed_value(context),
                 m34: self.m34.to_computed_value(context),
-                m41: self.m41.to_computed_value(context),
-                m42: self.m42.to_computed_value(context),
-                m43: self.m43.to_computed_value(context),
+                m41: length_to_computed_number(self.m41.to_computed_value(context)),
```

Comment:
Yeah, length values are getting normalized into numbers during computed value conversion. See:
```
data:text/html,<script>var div = document.createElement('div');div.style.MozTransform = "matrix(1, 1, 1, 1, 40px, 120px)";alert("Specified: " + div.style.MozTransform);alert("Computed: " + window.getComputedStyle(div).MozTransform);</script>
```
Otherwise interpolation algorithm will fail.